moon/skyline.py

```diff
@@ -0,0 +1,100 @@
+"""
+  Moon Skyline analysis
+"""
+import math
+
+import numpy as np
+import cv2  # for debug
+
+
+def skyline(img):
+    b = img[:,:,0]
+    g = img[:,:,1]
+    r = img[:,:,2]
+    mask = r > 10
+    return mask.argmax(axis=0)
+
+
+def find_peaks(arr):
+    x = arr.argmin()
+    y = arr[x]
+    ret = []
+    if x > 0:
```

Comment:
Why do you treat 0 as a special case?

Reply:
well, actually both areas near the edges are not reliable - you cannot tell if it is really **peak** ... it could be the edge, which is not good enough for localization

Reply:
p.s. you are right, that this `if` is bad because if the horizon is really tilted then you detect this first maxima on the border and the rest is ignored.